openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -373,6 +373,8 @@ private void stopSharing(Client c, String uid) {
 		StreamDesc sd = doStopSharing(c.getSid(), uid);
 		if (sender != null && sd != null) {
 			sender.stopBroadcast(this);
+		} else {
```

Comment:
this usually happens on `double-stop` are you sure it will produce ghosts?

Reply:
The stop doesn't directly. But it does indirectly. Cause when its stopped like that on startingRecording again, some stream object isn't 100% cleared. Which then leads to a ghost KStream and a failed restart of the recording.